README.md

```diff
@@ -34,10 +34,10 @@ Let a valid deployment URN be:
 urn:decentraland:entity:bafkreihpipyhrt75xyquwrynrtjadwb373xfosy7a5rhlh5vogjajye3im
 ```
 
-That deployment will be downloaded from the configured content server by default. But for testing purposes, the content servers are not always the most straight forward way to test. To help the operations, a baseUrl query parameter can be added: `?baseUrl=https://worlds-content-server.decentraland.org/ipfs/` yielding a full URN like this:
+That deployment will be downloaded from the configured content server by default. But for testing purposes, the content servers are not always the most straight forward way to test. To help the operations, a baseUrl query parameter can be added: `?baseUrl=https://worlds-content-server.decentraland.org/contents/` yielding a full URN like this:
 
 ```
-urn:decentraland:entity:bafkreihpipyhrt75xyquwrynrtjadwb373xfosy7a5rhlh5vogjajye3im?baseUrl=https://worlds-content-server.decentraland.org/ipfs/
+urn:decentraland:entity:bafkreihpipyhrt75xyquwrynrtjadwb373xfosy7a5rhlh5vogjajye3im?baseUrl=https://worlds-content-server.decentraland.org/contents/
```

Comment:
They both work. Are we "deprecating" `/ipfs` one?

Reply:
Unless there is a plan to change everything to /ipfs/ I'd vote for keeping consistency across content servers for the time being